### PR TITLE
Resolves issue #3

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- Integrate the Travis CI build.
- Resolves issue #3.
- The Travis CI build log is available [here](https://travis-ci.org/open-source-contributions/Python-Random-Password-Generator/builds/567826761).
- Add usage about generating password via executing `run.py`.

@ismailtasdelen, please review this PR if possible. Thanks.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+setup(name='rpg',
+      version='0.1',
+      description='Python - Random Password Generator ( R.P.G. )',
+      url='https://bitbucket.org/iclk4ami/cim-db-api',
+      author='İSMAİL TAŞDELEN',
+      author_email='pentestdatabase@gmail.com',
+      license='MIT',
+      packages=['source'],
+      python_requires=">=3",
+      zip_safe=False)
```
Comment (peter279k) [inline setup.py]: Initialize the `setup.py` let this `RPG` be a Python package.

File: source/rpg.py [hunk 2]
```diff
@@ -4,16 +4,16 @@
 import string
 import random
 
-def random_password_genertor():
+def random_password_generator():
```
Comment (peter279k) [inline source/rpg.py]: Fix typo.

File: source/rpg.py [hunk 3]
```diff
@@ -4,16 +4,16 @@
...
     chars = string.ascii_uppercase + string.ascii_lowercase + string.digits
     size = 8
-    return ''.join(random.choice(chars) for x in range(size,20))
+    return ''.join(random.choice(chars) for x in range(size, 20))
 
-def random_password_genertor_ico():
-	random_password_genertor_ico = """
+def random_password_generator_ico():
+	random_password_generator_ico = """
```
Comment (peter279k) [inline source/rpg.py]: Fix typo.

File: tests/rpg_test.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# -*- coding:utf-8 -*-
```
Comment (peter279k) [inline tests/rpg_test.py]: Initialize `rpg_test` with `pytest` tool.

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
```
Comment (peter279k) [inline .travis.yml]: The Python `2.x`  will be deprecated in 2020.

It's only add these versions to do unit tests during Travis CI build.

**Comment by ismailtasdelen** [comment]
I understand better now. And I'm grateful for that.

**Comment by peter279k** [comment]
Hi @ismailtasdelen. Thanks for your concern.

If possible, you can also enable the Travis CI build and add the Travis CI badge in the `README`.

**Comment by ismailtasdelen** [comment]
Of coruse @peter279k 👍

**Pull request merged**